conformance/utils/kubernetes/helpers.go

```diff
@@ -106,7 +106,7 @@ func NamespacesMustBeReady(t *testing.T, c client.Client, namespaces []string, s
 				t.Errorf("Error listing Gateways: %v", err)
 			}
 			for _, gw := range gwList.Items {
-				if !findConditionInList(t, gw.Status.Conditions, "Ready", "True") {
+				if !findConditionInList(t, gw.Status.Conditions, "Ready", "True", "Ready") {
```

Comment:
Same comment as above, would prefer "" instead of "Ready" as the last arg. (If I understand that code, that will allow any reason).

Reply:
I changed this to the empty string, with a comment.